draft-ietf-rtcweb-jsep.xml

```diff
@@ -2840,8 +2840,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <xref target="RFC4145"></xref>, Section 4, and clarified
               for use in DTLS-SRTP scenarios in
               <xref target="RFC5763"></xref>, Section 5. The role value
-              in the answer MUST be "active" or "passive"; the "active"
-              role is RECOMMENDED.</t>
+              in the answer MUST be "active" or "passive". When
+              the the offer contains the "actpass" value, as
+              will always be the case with JSEP endpoints,
+              the answerer SHOULD use the "active" role.
```

Comment:
But, this isn't always the right guidance, since in a re-offer scenario the value may need to be either active or passive.